UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs

```diff
@@ -509,12 +509,25 @@ public void CmdRemoveBounty(int index, bool completeBounty, NetworkConnectionToC
 			CargoManager.Instance.ActiveBounties.Remove(CargoManager.Instance.ActiveBounties[index]);
 		}
 
+		[Command(requiresAuthority = false)]
```

Comment:
correct me if I am wrong, but if ``requiresAuthority`` is false then it means another player could use an admin player object to send this command. Why is it needed?